pkg/federation/sdlmerge/sdlmerge_test.go

```diff
@@ -12,6 +12,8 @@ import (
 	"github.com/jensneuse/graphql-go-tools/pkg/operationreport"
 )
 
+var N = &normalizer{}
```

Comment:
I really don't think that `N` is a good name, even for test files only

Reply:
OK. I will rename it.